docs/architecture.md

```diff
@@ -21,52 +21,52 @@ This leads to constant development and DevOps/MLOps work.
 
 Furthermore, as your project scales, you need greater computation power or GPUs, and you need to access large-scale data sets.
 This cannot work on laptops.
-You need a way to seamlessly run your code on a remote cluster and automatically scale it out.
+You need a way to seamlessly run your code on a remote cluster and automatically scale it.
```

Comment:
REJECT Please revert this change. "scale out" and "scale up" are known technical terms and I believe the original writer intentionally used "out" here to refer to scaling out (as opposed to scaling up).